.github/workflows/markdownlint.yml

```diff
@@ -0,0 +1,34 @@
+---
+# yamllint disable rule:line-length
```

Comment:
Is this needed?

(If I can I don't disable a linter rule.)

Reply:
I actually do not know, I copy-pasted this from other templates. @richm could you provide some info around this?

Reply:
If we are using this in all `.github` files (or non-ansible related files), than we should keep it for consistency.